docs/explanations/structuring_your_project.md

```diff
@@ -0,0 +1,99 @@
+
+> Note: This page is a work in progress and will contain code examples at a later date.
+
+## Permissions Management
+
+We recommend creating Synapse Teams for permission management on your Synapse project.  These are the recommended teams to create:
+
+* **<project> Admin** - This team should have "Administrator" access to the project.  This team should be used to manage the project and grant access to other teams.
+
+* **<project> Users** - Optional: If you want to grant all registered users on Synapse download access to your project, you can click the "Make Public" button in project's sharing settings.  This team should have "Can Download" access to the project.  This team should be used to grant download access to the project.
+
+Below are some key permission criterias to consider when setting up your project:
+
+* Create a Synapse Team and Project per data contributor if there are (1) multiple data contributors and (2) the data contributors should not have access to each other's raw data. You would then create a central "public" project that will contain the harmonized data. You can technically leverage local share settings by creating private folders in one project but managing local share settings is more complicated and not recommended.
+* Do not mix data that requires different permission models within a folder. For example, if you have a folder that contains both public and private data, you should create two folders, one for public data and one for private data.  You can then grant the appropriate permissions to each folder. You can use local share settings to manage each file's permission, but this is not recommended!
+
+
+## Project Structure
+
+This document will help you understand how to organize your data in Synapse to make it easier for the community to find and use your data.
+
+When organizing your data for upload we have a preferred organization (flattened data layout) and an alternate option (hierarchy data layout) if your project requires that.
+
+> NOTE: If you and your contributing site decide to use a hierarchical file structure within your cloud storage location, please remember that each top-level folder and all of its subfolders must contain data of the same type (see details below).
+
+
+### Top Level Folder Names
+
+Top level folders correspond to the datasets being submitted. See the examples below. You can name your datasets in a way that is descriptive for your contributing site.
+
+You can use either the Hierarchy or Flattened data layout according to the examples below. In the hierarchical case, you would fill in one manifest and include all files in experiment/batches; in the flattened case, you would fill in one manifest for each top level folder.  The manifest would be Synapse annotations which can be used to query the data when a File View is created.
```

Comment:
You reference a `manifest` here and in the file structure as CSV files. I know we have https://python-docs.synapse.org/explanations/manifest_tsv/ - which is a TSV file.

https://help.synapse.org/docs/Managing-Custom-Metadata-at-Scale.2004254976.html#ManagingCustomMetadataatScale-BatchUploadFileswithAnnotations as references this as a TSV.

Is CSV supported? Do we have documentation to link off to to explain what is contained within these files?